src/main/scala/mesosphere/marathon/state/Timestamp.scala

```diff
@@ -15,6 +15,8 @@ case class Timestamp(dateTime: DateTime) extends Ordered[Timestamp] {
     case _               => false
   }
 
+  override def hashCode: Int = time.hashCode
```

Comment:
Could you add a comment to make it explicit that we depend upon `time` being a UTC DateTime to uphold the `equals`/`hashCode` contract?